packages/app-content-pages/next.config.js

```diff
@@ -10,7 +10,7 @@ const assetPrefixes = {
   branch: 'https://fe-project-branch.preview.zooniverse.org/about',
   staging: 'https://frontend.preview.zooniverse.org/about',
```

Comment:
Should the `staging` prefix be deleted similar to app-project, too?

Reply:
I don't think it's doing anything as is, but I suppose for consistency sake?

Reply:
I should have updated #4238 accordingly

Reply:
Agreed, I'll push an update